shell/src/main/java/org/apache/accumulo/shell/commands/ScriptCommand.java

```diff
@@ -68,6 +74,9 @@
   public int execute(String fullCommand, CommandLine cl, Shell shellState) throws Exception {
 
     boolean invoke = false;
+
+    Shell.log.warn(
+        "Deprecated -- much of the use case for having this script command is likely able to be replaced by Java 11's JShell feature");
```

Comment:
This wording, and the wording of the above javadoc, is a big ambiguous/uncertain. That probably reflects the original ticket's uncertainty. However, if the original ticket has been thoroughly investigated and is ready for a code change, then we should have some certainty in the final wording. I recommend:

```suggestion
    Shell.log.warn("The script command is deprecated; use jshell for scripting instead");
```